tests/MyTests.cpp

```diff
@@ -38,21 +38,40 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include <limits>
 #include <type_traits>
 
-#include "Exception.hpp"
-#include "MyLibrary.hpp"
+#include "../src/Exception.hpp"
+//#include "../src/MyLibrary.hpp"
+#include <iostream>
+#include <dcmtk/dcmpstat/dcmpstat.h>
+#include "../src/metadataEditor.cpp"
+#include "../src/metadataEditor.hpp"
 
 using namespace cpp_template;
 
 // This tests the output of the `get_nth_prime` function
 TEST_CASE("correct primes are returned", "[primes]") {
-  CHECK(get_nth_prime(0) == 2);
-  CHECK(get_nth_prime(1) == 3);
-  CHECK(get_nth_prime(2) == 5);
-  CHECK(get_nth_prime(854) == 6619);
+
+  DcmFileFormat image;
+  OFCondition status = image.loadFile("../DICOM_Images/1-1copy.dcm");
+  
+if (status.good())
+{
+  OFString patientName;
+  if (image.getDataset()->findAndGetOFString(DCM_PatientName, patientName).good())
+  {
+    std::cout << "Patient's Name: " << patientName << std::endl;
```

Comment:
rather than printing the info for manual checking, need to put some automated tests in here (i.e some `CHECK` statements)